src/fu-release.c

```diff
@@ -863,7 +863,7 @@ fu_release_load(FuRelease *self,
 	if (tmp != NULL)
 		fwupd_release_set_remote_id(FWUPD_RELEASE(self), tmp);
 	tmp = xb_node_query_text(component, "../custom/value[@key='LVFS::Distributor']", NULL);
-	if (g_strcmp0(tmp, "community") == 0)
+	if (g_strrstr(tmp, "community") != NULL)
```

Comment:
I thought the idea was a _prefix_? i.e. `if (tmp != NULL && g_str_has_prefix(tmp, "community") == 0)`

Reply:
Oh right, my glib is a bit rusty lately :sweat_smile: